src/common/executor.ts

```diff
@@ -78,4 +87,34 @@ export class Executor {
             outputPane.appendLine(value);
         }
     }
+
+    private static createTerminal(terminal: string): vscode.Terminal {
+        const envVars  = Executor.getEnvFromConfig();
+        const options: vscode.TerminalOptions = { name: terminal };

```

Comment:
Would `const options: vscode.TerminalOptions = { name: terminal, env:  envVars};` be simpler?

Reply:
Same as above.

Reply:
Per definition, env parameter is `Object with environment variables that will be added to the VS Code process.` It is added values, it would not affect other environment variables

Reply:
Thanks, got it. To make it consistent in the code as subprocess scenario. I prefer to add only valid key which is more safer.